app/models/admin_menu.rb

```diff
@@ -77,8 +87,14 @@ def self.nested_menu_items(scope_name, nav_item)
 
       items[:children].each do |child|
         return items if child[:controller] == nav_item
+        # NOTE: [@jeremyf] trying to puzzle this one out.  My read is that if the "grandchild" of the
```

Comment:
I suspect a regression occurred in the last couple of months. Here's some more detail: https://www.loom.com/share/7e0ce358120e4f3fb6df59f2cfad6183

It seems to not be a regression on your PR, so either you or I can look into this in a separate PR.

Reply:
That Loom is super helpful.  I'm prepared to dive deeper into the AdminMenu to resolve this regression.  I have some ideas and with this context, I am well equipped to dig deeper.

I'm going to write up an issue (and link to your Loom).  This way we can merge this PR and also not lose track of fixing that regression.

Reply:
Sounds good, thanks Jeremy!

Reply:
I added https://github.com/forem/forem/issues/16901 to capture this regression.